kymatio/scattering1d/core/timefrequency_scattering.py

```diff
@@ -268,3 +282,45 @@ def frequency_scattering(X, backend, filters_fr, oversampling_fr,
         # and n=(n2, n_fr) if spinned. This 'n' tuple is unique.
         yield {**X, 'coef': Y_fr, 'n': (X['n'][1:] + (n_fr,)),
             'j_fr': j_fr, 'n_fr': n_fr, 'spin': spin}
+
+
+def time_averaging(U_2, backend, phi_f, oversampling):
+    """
+    Parameters
+    ----------
+    U_2 : dictionary with keys 'coef' and 'j', typically returned by
+        frequency_scattering
+    backend : module
+    phi_f : dictionary. Temporal low-pass filter in Fourier domain,
+        same as scattering1d
+    oversampling : int >=0
+        Yields scattering coefficients at the sample
+        rate max(1, 2**(log2_T-oversampling)). Hence, raising oversampling by
+        doubles the sample rate, until reaching the native sample rate.
```

Comment:
- `Raising oversampling by doubles` => missing `one`.

- `max(1, 2**(log2_T-oversampling))` is the number of samples per stride rather than "sample rate"? Raising oversampling by one doubles the sample rate but halves `2**(log2_T-oversampling)`.

Reply:
thank you. you're right